config/default/manager_default_images.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         - name: RELATED_IMAGE_IRONIC_CONDUCTOR_IMAGE_URL_DEFAULT
           value: quay.io/podified-antelope-centos9/openstack-ironic-conductor:current-podified
         - name: RELATED_IMAGE_IRONIC_INSPECTOR_IMAGE_URL_DEFAULT
-          value: quay.io/podified-antelope-centos9/openstack-ironic-inspector:current-podified
+          value: quay.io/mschuppe/openstack-ironic-inspector:inspector-httpd
```

Comment:
needs to be reverted when the image got updated

Reply:
I'll submit a revert for this as soon its promoted to quay.io/podified-antelope-centos9/openstack-ironic-inspector:current-podified . its not there yet